public/locales/en/translation.json

```diff
@@ -1,4 +1,8 @@
 {
+  "subheaderButton": "Apply for your UI benefits on UI Online",
+  "subheaderH2": "Applying to Unemployment Insurance",
```

Comment:
nit, I think it's better to use subheaderTitle / Subtitle or something like that, so it isn't confusing / doesn't need to be changed if the H2 tag changes?